### PR TITLE
added another quote

### PR DESCRIPTION
**Pull request by srujan0404**
**Description**
Another quote to capture about Meshery on layer5.io

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
Oh, no. The site build and preview is failing...

File: src/sections/Pricing/review-slider.js [hunk 1]
```diff
@@ -185,12 +187,16 @@ const Reviews = () => {
           />
           <Customers
             type="3"
+            quote="Visualization is a great idea for complex environments!"
+            person="Patrick Steinig"
+            title="Platform Engineering, HDI Group"
+            image={patrick}
```
Comment (leecalcote) [inline src/sections/Pricing/review-slider.js:193]:
```suggestion
            image={Patrick}
```

File: src/sections/Pricing/review-slider.js [hunk 2]
```diff
@@ -15,9 +15,11 @@ import Mrittika from "./reviews/mrittika-ganguli.jpeg";
 import Liz from "./reviews/liz-rice.jpeg";
 import Ala from "./reviews/ala-eddine-benhassir.jpeg";
 import Alex from "./reviews/alex-hokanson.jpeg";
+import patrick from "./reviews/patrick-steinig.jpg";
```
Comment (leecalcote) [inline src/sections/Pricing/review-slider.js:18]:
```suggestion
import Patrick from "./reviews/patrick-steinig.jpg";
```

**Comment by leecalcote** [comment]
Update made...

**Comment by leecalcote** [comment]
@srujan0404 the build preview isn't completing...

**Comment by srujan0404** [comment]
hey @leecalcote, I am not able to identify the correct reason but I think this issue to related to the image format

**Comment by vishalvivekm** [comment]
@srujan0404 Let's discuss this on Websites's call today 6:30 PM IST. Let's add this as an agenda item into the meeting minutes

**Comment by l5io** [comment]
🚀 Preview for commit a6e75e7702881ae4309206459c5047468a6c1361 at: https://65f739e99a02abbe241c97f5--layer5.netlify.app

**Pull request merged**